test/realm_test.dart

```diff
@@ -400,11 +400,9 @@ Future<void> main([List<String>? args]) async {
 
       expect(car.make, equals('Tesla'));
 
-      expect(() {
-        realm.write(() {
-          car.make = "Audi"; // setting late final is a runtime error :-/
-        });
-      }, throws<RealmUnsupportedSetError>());
+      realm.write(() {
```

Comment:
this one should still fail

Reply:
this one fixes this https://github.com/realm/realm-dart/pull/253 consider @desistefanova could you please revert this test change

Reply:
I guess we need to move #253 first and then rebase this one so we are sure the tests behaves the same

Reply:
> this one should still fail

Expected exception is reverted with https://github.com/realm/realm-dart/pull/248/commits/8c240e43b05bee6712223a5588ff8e65d4e02ac6.

Reply:
> I guess we need to move #253 first and then rebase this one so we are sure the tests behaves the same

I did rebase onto #253 and regenerated all the models again. 
I removed `final` from Lists also.